src/main/java/gregtech/api/metatileentity/implementations/MTECable.java

```diff
@@ -263,6 +266,127 @@ public void onPostTick(IGregTechTileEntity aBaseMetaTileEntity, long aTick) {
+                    for (int i = 0; i < aPlayer.inventory.mainInventory.length; i++) {
+                        ItemStack slot = aPlayer.inventory.mainInventory[i];
+                        if (slot != null && slot.getItem() == oldCable.getItem()
+                            && slot.getItemDamage() == oldCable.getItemDamage()
+                            && slot.stackSize < slot.getMaxStackSize()) {
+                            slot.stackSize++;
+                            addedToInventory = true;
+                            break;
+                        }
+                    }
+
+                    // If couldn't stack, try to find empty slot
+                    if (!addedToInventory) {
+                        addedToInventory = aPlayer.inventory.addItemStackToInventory(oldCable);
+                    }
+
+                    // If still couldn't add, drop in world
+                    if (!addedToInventory) {
+                        aPlayer.dropPlayerItemWithRandomChoice(oldCable, false);
+                    }
+                }
+
+                // Use one cable from player's hand
+                handItem.stackSize--;
+                if (handItem.stackSize <= 0) {
+                    aPlayer.inventory.setInventorySlotContents(aPlayer.inventory.currentItem, null);
+                }
+            }
+            // After setting new cable but before inventory handling
+            if (oldAmperage != handCable.mAmperage || oldVoltage != handCable.mVoltage) {
+                StringBuilder message = new StringBuilder();
+
+                // Amperage change
+                if (oldAmperage != handCable.mAmperage) {
+                    message.append(oldAmperage)
+                        .append("A → ")
+                        .append(handCable.mAmperage > oldAmperage ? EnumChatFormatting.GREEN : EnumChatFormatting.RED)
+                        .append(handCable.mAmperage)
+                        .append("A")
+                        .append(EnumChatFormatting.RESET);
+                }
+
+                // Separator if both changed
+                if (oldAmperage != handCable.mAmperage && oldVoltage != handCable.mVoltage) {
+                    message.append(" | ");
+                }
+
+                // Voltage change
+                if (oldVoltage != handCable.mVoltage) {
+                    message.append(oldVoltage)
+                        .append("V → ")
+                        .append(handCable.mVoltage > oldVoltage ? EnumChatFormatting.GREEN : EnumChatFormatting.RED)
+                        .append(handCable.mVoltage)
+                        .append("V")
+                        .append(EnumChatFormatting.RESET);
+                }
+
+                // Send the message using GT's utility method
+                GTUtility.sendChatToPlayer(
+                    aPlayer,
+                    StatCollector.translateToLocal("GT5U.item.cable.swapped") + message.toString());
```

Comment:
Fixes spacing:
![image](https://github.com/user-attachments/assets/2ce34946-d60b-4c12-aa75-37929965f66d)

```suggestion
                    StatCollector.translateToLocal("GT5U.item.cable.swapped") + "" + message.toString());
```